bin/truffleruby

```diff
@@ -16,18 +16,6 @@ if [ -n "$RUBY_BIN" ]; then
     exec "$RUBY_BIN" "$@"
 fi
 
-function mtime {
-    uname_str=$(uname)
-    if [ "$uname_str" = 'Linux' ] || [ "$uname_str" = 'SunOS' ]; then
-        stat -c '%Y' "$1" 2>/dev/null || echo 0
-    elif [ "$uname_str" = 'Darwin' ]; then
-        stat -f '%m' "$1" 2>/dev/null || echo 0
-    else
-        echo "unknown platform $uname_str" 1>&2
-        exit 1
-    fi
-}
```

Comment:
I think it's best to keep functions at the top, and not conditionally define them, particularly in the middle of the logic.